pkg/cmd/openshift-tests/run/options.go

```diff
@@ -73,14 +73,21 @@ func (o *RunSuiteOptions) Run(ctx context.Context) error {
 	k8simage.Init("")
 
 	// TODO the gingkoRunSuiteOptions needs to have flags then calculated options to express specified versus computed values
+	stabilitySetting := o.GinkgoRunSuiteOptions.ClusterStabilityDuringTest
```

Comment:
`o.GinkgoRunSuiteOptions.ClusterStabilityDuringTest` takes priority

Reply:
would you mind if I change the default of the cmd options to be "" instead? that way we can treat it as a real override from the release repo and otherwise take standard if nothing is suite defined.